### PR TITLE
fix (build): build search.go for all architectures except amd64

### PR DESCRIPTION
**Pull request by maddie**
Fix build for mips64 and mips64le

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/ristretto/257)

### CODE REVIEW

File: z/simd/search.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// +build 386 arm armbe arm64 mips mipsle mips64p32 mips64p32le ppc ppc64 ppc64le sparc
+// +build 386 arm armbe arm64 mips mipsle mips64p32 mips64p32le ppc ppc64 ppc64le sparc mips64 mips64le
```
Comment (atetubou) [inline z/simd/search.go:1]:
```suggestion
// +build !amd64
```

I think we don't need to enumerate all non-amd64 archs here.

**Comment by atetubou** [comment]
Thanks, and you can update the title of this PR as well?

**Review by atetubou — APPROVED** [review]

**Comment by maddie** [comment]
> Thanks, and you can update the title of this PR as well?

Done.

**Comment by atetubou** [comment]
@NamanJain8 could you take a look and merge this?

**Pull request merged**